### ISSUE TITLE
Should workflows check for repo name before running?

### ISSUE THREAD

**Issue by pllim** [astropy/repo_stats#3]
Especially the workflows that generate artifacts, we don't want any fork to be accidentally running them right if they have Actions turned on for whatever reason? See https://github.com/astropy/astropy/blob/54578767c44950710562c1333bfe1a5312a083e3/.github/workflows/ci_cron_weekly.yml#L34 for example.

```yaml
if: (github.repository == 'astropy/repo_stats')
```

**Comment by jeffjennings** [comment]
Yes definitely, thank you for thinking of that, it hadn't occurred to me.
